lib/ObisNames.js

```diff
@@ -6,352 +6,352 @@
+        135: "SLP Monday",
+        136: "SLP Tuesday",
+        137: "SLP Wednesday",
+        138: "SLP Thursday",
+        139: "SLP Friday",
+        140: "SLP Saturday",
+        141: "SLP Working day",
+        142: "SLP Holiday",
+        143: "Currency",
+        144: "Currency prediction",
+        145: "CO2",
+        146: "CO2 Prediction"
     },
     'de': {
         // 1-127: Messgerätespezifischer Kanal
-        128: 'Einsparung',
-        129: 'Sollwert',
-        130: 'Prognose',
-        131: 'Maximale Grenzwerte Arbeitstag',
-        132: 'Maximale Grenzwerte Nichtarbeitstag',
-        133: 'Minimale Grenzwerte Arbeitstag',
-        134: 'Minimale Grenzwerte Nichtarbeitstag',
-        135: 'SLP Montag',
-        136: 'SLP Dienstag',
-        137: 'SLP Mittwoch',
-        138: 'SLP Donnerstag',
-        139: 'SLP Freitag',
-        140: 'SLP Samstag',
-        141: 'SLP Arbeitstag',
-        142: 'SLP Nichtarbeitstag',
-        143: 'Geldbetrag',
-        144: 'Prognose Geldbetrag',
-        145: 'Emmision CO2',
-        146: 'Prognose Emmision CO2'
+        128: "Einsparung",
+        129: "Sollwert",
+        130: "Prognose",
+        131: "Maximale Grenzwerte Arbeitstag",
+        132: "Maximale Grenzwerte Nichtarbeitstag",
+        133: "Minimale Grenzwerte Arbeitstag",
+        134: "Minimale Grenzwerte Nichtarbeitstag",
+        135: "SLP Montag",
+        136: "SLP Dienstag",
+        137: "SLP Mittwoch",
+        138: "SLP Donnerstag",
+        139: "SLP Freitag",
+        140: "SLP Samstag",
+        141: "SLP Arbeitstag",
+        142: "SLP Nichtarbeitstag",
+        143: "Geldbetrag",
+        144: "Prognose Geldbetrag",
+        145: "Emmision CO2",
+        146: "Prognose Emmision CO2"
     },
     "nl": {
         // 1-127: Meter-specifiek kanaal
-        128: "Sparen",
-        129: "Setpoint",
+        128: "Besparing",
+        129: "Ingestelde waarde",
         130: "Voorspelling",
-        131: "Maximale limieten werkdag",
-        132: "Maximumgrenzen niet-werkdag",
-        133: "Minimale limieten werkdag",
-        134: "Minimumgrenzen niet-werkdag",
-        135: 'SLP maandag',
-        136: 'SLP Dinsdag',
-        137: 'SLP Woensdag',
-        138: 'SLP Donderdag',
-        139: 'SLP Vrijdag',
-        140: 'SLP zaterdag',
-        141: "SLP Werkdag",
-        142: "SLP-niet-werkdag",
+        131: "Maximale grenswaarden werkdag",
+        132: "Maximale grenswaarden rustag",
+        133: "Minimale grenswaarden werkdag",
+        134: "Minimale grenswaarden rustdag",
+        135: "SLP maandag",
+        136: "SLP dinsdag",
+        137: "SLP woensdag",
+        138: "SLP donderdag",
+        139: "SLP vrijdag",
+        140: "SLP zaterdag",
+        141: "SLP werkdag",
+        142: "SLP rustdag",
         143: "Geldbedrag",
-        144: "Voorspelde hoeveelheid geld",
-        145: 'Emmision CO2',
-        146: "Voorspelling Emmision CO2"
+        144: "Prognose geldbedrag",
```

Comment:
Currency Prediction : Lijk me eerder Valuta voorspelling ?

Reply:
Valuta =algemeen geldig betaalmiddel in een bepaald land (Van Dale). Deze vertaling past ook niet.

Ik vind overigens 'prognose geldbedrag' ook niet mooi. Ik zat eerst te denken aan opbrengstprognose, maar in dat woord zit niet dat het om een geldbedrag gaat (het zou ook om kWh kunnen gaan). We zouden er van kunnen maken 'opbrengstprognose in euro's'.